test/File/FileInformationTraitTest.php

```diff
@@ -0,0 +1,286 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-validator for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-validator/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Validator\File;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Http\Message\StreamInterface;
+use Psr\Http\Message\UploadedFileInterface;
+use ZendTest\Validator\File\TestAsset\FileInformation;
+use Zend\Validator\Exception\InvalidArgumentException;
+
+/**
+ * @group      Zend_Validator
+ */
+class FileInformationTraitTest extends TestCase
+{
+    /** @var ObjectProphecy */
```

Comment:
It is the correct type, but very useful here would be to add also `|StreamInterface`